This patch is needed for newer automake to complete correctly. Without this
patch it returns an error as:

  xineplug_dmx_image.la is already to be installed in lib directory.
Index: xine-lib-1.1.2cvs20060328/src/demuxers/Makefile.am
===================================================================
--- xine-lib-1.1.2cvs20060328.orig/src/demuxers/Makefile.am
+++ xine-lib-1.1.2cvs20060328/src/demuxers/Makefile.am
@@ -19,10 +19,7 @@ if HAVE_LIBMNG
 mng_module = xineplug_dmx_mng.la
 endif
 
-if HAVE_WAND
-image_module = xineplug_dmx_image.la
-endif
-if HAVE_GDK_PIXBUF
+if BUILD_DMX_IMAGE
 image_module = xineplug_dmx_image.la
 endif
 
Index: xine-lib-1.1.2cvs20060328/configure.ac
===================================================================
--- xine-lib-1.1.2cvs20060328.orig/configure.ac
+++ xine-lib-1.1.2cvs20060328/configure.ac
@@ -2251,6 +2251,8 @@ dnl as long as neither autoconf nor auto
 dnl check we will have to call it
 _AM_DEPENDENCIES([OBJC])
 
+AM_CONDITIONAL([BUILD_DMX_IMAGE], [test x"$have_imagemagick" = "xyes" -o x"$no_gdkpixbuf" != "xyes"])
+
 
 dnl ---------------------------------------------
 dnl Output configuration files
